java/main/com/google/privacy/differentialprivacy/BoundedSum.java

```diff
@@ -124,10 +125,38 @@ public double computeResult() {
 
     resultReturned = true;
     double lInfSensitivity =
-        getLInfSensitivity(params.lower(), params.upper(), params.maxContributionsPerPartition());
-    return params
-        .noise()
-        .addNoise(sum, getL0Sensitivity(), lInfSensitivity, params.epsilon(), params.delta());
+            getLInfSensitivity(params.lower(), params.upper(), params.maxContributionsPerPartition());
+    noisedSum =
+            params
+                  .noise()
+                  .addNoise(sum, getL0Sensitivity(), lInfSensitivity, params.epsilon(), params.delta());
+    return noisedSum;
+  }
+
+  // ComputeConfidenceInterval computes a confidence interval that contains the true count with
+  // a probability greater or equal to 1 - alpha using the noised sum computed by computeResult().
+  public ConfidenceInterval computeConfidenceInterval(double alpha) {
+    if (!resultReturned) {
+      throw new IllegalStateException("Noised sum must be computed before calling this function.");
+    }
+    ConfidenceInterval confInt =
+            params
+                    .noise()
+                    .computeConfidenceInterval(
+                            noisedSum,
+                            params.maxPartitionsContributed(),
+                            (double) params.maxContributionsPerPartition(),
+                            params.epsilon(),
+                            params.delta(),
+                            alpha);
+    if ((params.lower() < 0) && (params.upper() <= 0)) {
```

Comment:
if (params.upper() <= 0.0) {

the lower bound is implicitly less than 0 in this case

also, it is good style to write mark double falues as such, e.g. by writing zero as 0.0